add cloud foundry deployment setup (#1)
anynines/anynines-react-boilerplate: README.md

```

## Cloud Foundry Deployment

> You can deploy your react-boilerplate on the [Cloud Foundry Platform](https://www.cloudfoundry.org/) [anynines](https://paas.anynines.com/). For a Cloud Foundry deployment you need to install the Cloud [Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html). 

Reviewer: ```suggestion
You can deploy your react-boilerplate on the [Cloud Foundry Platform](https://www.cloudfoundry.org/) [anynines](https://paas.anynines.com/). For a Cloud Foundry deployment you need to install the [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html). 
```

> You can deploy your react-boilerplate on the [Cloud Foundry Platform](https://www.cloudfoundry.org/) [anynines](https://paas.anynines.com/). For a Cloud Foundry deployment you need to install the Cloud [Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html). 

Reviewer: ```suggestion
You can deploy your react-boilerplate on any [Cloud Foundry Platform](https://www.cloudfoundry.org/) like [anynines](https://paas.anynines.com/). For a Cloud Foundry deployment you need to install the Cloud [Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html). 
```

Reviewer: I don't know about this one, but I think a little spacing with using `like` between `Cloud Foundry Platform` and `anynines` helps to differentiate that there are two separate links.

Specify your Cloud Foundry API endpoint and authenticate with your account.

```bash
cf api [URL_FOR_API_ENDPOINT]
cf login
```

> You can select or change the traget for your app with:

Reviewer: ```suggestion
You can select or change the target for your app with:
```

```bash
cf target -o [ORGANISATION] -s [SPACE]
```

> Add the name of your app for the placeholder `[YOUR_APP_NAME]` in the `manifest.yml` and `deploy.sh` or `deploy.bat` file.

Reviewer: ```suggestion
Replace `[YOUR_APP_NAME]` with the name of your app in the `manifest.yml` and respectively in the `deploy.sh` or the `deploy.bat` file if you are using a unix-like or windows environment.
```

> Run the corresponding script for your OS to deploy your app.

Reviewer: ```suggestion
To start deploying the app you just have to run the corresponding deploy script for your environment. 
```

```bash

> deploy.sh

Reviewer: Just an idea: How cool would it be to have to just run the `deploy.sh` and it interactively asks you about your api, login, app name, etc? Like an included bootstrap cli that helps to setup a first time deployment.

eg
```sh
$ ./deploy.sh
$ react-boilerplate: First time setup running
$ Please specifiy your API endpoint.
> https://api...
$ react-boilerplate:  You are currently not logged-in
> user@example.com
> pass****
$ react-boilerplate: The manifest is not configured yet. Please specify your app-name
> my-app
...
```

I don't know if this is actually out of scope, but I personally think such a small script would be quite fancy 😄 Or even keep them split and use it as a seperate setup script

Author: I like the idea. But i think we should put this idea in another ticket or discuss this idea with the frontend team :)
